modules/ROOT/partials/configuration/rtc_client_connected.adoc

```diff
@@ -9,12 +10,13 @@ To help with generating a user interface for connected users, an object for user
 ifeval::[{plugincode} != "rtc"]
```

Comment:
this looks like a string comparison, theres a few of them, should be:

["{plugincode}" != "rtc"]

Reply:
can you also confirm if the != not logic is used correctly here? It could be an old code bug that was masked by the not comparison

Reply:
Something has gone wrong with the merge Hamza did recently here by the looks, as this isn't a file in `advtable`. In this particular case Hamza has fixed that in https://github.com/tinymce/tinymce-docs/pull/2309.

Reply:
Yeah something weird happened somewhere but it's fixed now